### PR TITLE
add specific title to each page

### LINKED ISSUE #180: add page title to different kinds of pages

### PR DESCRIPTION
**Pull request by hunglmtb**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>
> /kind api-change
> /kind backend-change
> /kind ui-change
> /kind bug
> /kind cleanup
> /kind design
> /kind documentation
> /kind failing-test
> /kind feature
> /kind flake

/kind ui-change

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #180

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: src/shared/common/address-link.tsx [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+import { Link } from "react-router-dom";
+
+type Props = {
+  address: string;
+  text: string;
+  // tslint:disable:no-any
+  location: any;
+};
+
+class AddressLinkInner extends Component<Props> {
```
Comment (puncsky) [inline src/shared/common/address-link.tsx]: it is not just address, right?

there are block hash, action hash, block height.

Comment (hunglmtb) [reply src/shared/common/address-link.tsx]: updated at 9e42a33 in PR #194

File: src/shared/action/action-detail.tsx [hunk 2]
```diff
@@ -33,6 +35,7 @@ class ActionDetailsInner extends PureComponent<Props> {
 
     return (
       <ContentPadding>
+        <Helmet title={`${t("action.action")} - ${t("meta.description")}`} />
```
Comment (puncsky) [inline src/shared/action/action-detail.tsx]: `IoTeX ${t("action.action")} - ${action hash}`

File: src/shared/address-details/address-details.tsx [hunk 3]
```diff
@@ -36,6 +40,7 @@ class AddressDetailsInner extends PureComponent<Props> {
     };
     return (
       <ContentPadding>
+        <Helmet title={`${t("address.address")} - ${t("meta.description")}`} />
```
Comment (puncsky) [inline src/shared/address-details/address-details.tsx]: `IoTeX ${t("address.address")} ${address}`

File: src/shared/action/action-detail.tsx (hunk 2 above)
Comment (puncsky) [reply src/shared/action/action-detail.tsx]: use the exact hash

File: src/shared/address-details/address-details.tsx (hunk 3 above)
Comment (puncsky) [reply src/shared/address-details/address-details.tsx]: use the exact address content

File: src/shared/block/block-detail.tsx [hunk 4]
```diff
@@ -49,6 +51,7 @@ class BlockDetailsInner extends PureComponent<Props, State> {
 
     return (
       <ContentPadding>
+        <Helmet title={`${t("block.block")} - ${t("meta.description")}`} />
```
Comment (puncsky) [inline src/shared/block/block-detail.tsx]:
```
IoTeX ${t("block.block")} ${block height}
```

**Pull request merged**